auth0/v3/management/jobs.py

```diff
@@ -51,8 +51,13 @@ def import_users(self, connection_id, file_obj):
             file_obj (file): A file-like object to upload. The format for
                 this file is explained in: https://auth0.com/docs/bulk-import
         """
+
+        # This conversion is necessary because the requests library will convert True to 'True'.
+        # The Management API only respects 'true'
+        upsert = 'true' if upsert else 'false'
```

Comment:
Instead of this line, could you use `str(upsert).lower()` in Line 60. Though there is nothing wrong with your approach, this is the preferred way of handling boolean parameters in the SDK. So we can stick to it for consistency.